charts/kubewarden-controller/README.md

```diff
@@ -29,14 +30,55 @@ $ helm install --wait --create-namespace -n kubewarden kubewarden-controller kub
 This will install cert-manager, and kubewarden-controller on the Kubernetes
 cluster in the default configuration (which includes self-signed TLS certs).
 
-The default configuration values should be good enough for the
-majority of deployments, all the options are documented
-[here](https://charts.kubewarden.io/#configuration).
+The default configuration values should be good enough for the majority of
+deployments. All the options are documented in the configuration section.
+
+## Upgrading the chart
+
+Please refer to the release notes of each version of the helm charts.
+These can be found [here](https://github.com/kubewarden/helm-charts/releases).
+
+## Uninstalling the chart
+
+To uninstall/delete the `kubewarden-controller` release use the following
+command:
+
+```console
+$ helm uninstall -n kubewarden kubewarden-controller
+```
 
-## Usage
+The command removes all the Kubernetes components associated with the chart, all
+policy serves and their policies, and deletes the release along with the release
```

Comment:
typo in serves
```suggestion
policy server and their policies, and deletes the release along with the release
```

Reply:
Thanks! fixed in https://github.com/kubewarden/helm-charts/pull/37/commits/1e825cb5b4c6222b68fd537bdd6aecaa444c8fda